dallinger/experiment_server/experiment_server.py

```diff
@@ -354,9 +359,12 @@ def advertisement():
         ad_string = insert_mode(ad_string, config.get('mode'))
         return render_template_string(
             ad_string,
+            is_sandbox=False,
```

Comment:
There are three modes: `debug`, `sandbox`, and `live`. maybe we should condense `is_sanbox` and `debug_mode` into a single parameter, `mode`, that takes one of the three string values.